mmpose/datasets/pipelines/topdown_transforms.py

```diff
@@ -219,7 +219,8 @@ def __repr__(self) -> str:
 
 @TRANSFORMS.register_module()
 class TopdownGenerateRegressionLabel(BaseTransform):
```

Comment:
Do you think it would be better to combine `TopdownGenerateHeatmap` and `TopdownGenerateRegressionLabel` into a  unified transform (e.g. `TopdownGenerateTarget`) or not?

Reply:
It seems to be a good idea :) I think it's better to fold them as two util functions and use `TopdownGenerateTarget` to call